data.json

```diff
@@ -53,6 +53,15 @@
     "dateAdded": "2019-10-07T13:05:05.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "AquaSec",
+    "difficulty": "medium",
+    "description": "In honor of Hacktoberfest, we want to recognize our community and reward your contributions to our open source projects during October.",
```

Comment:
Can you update the description please?

Reply:
Url should be fixed and description has been updated. Thanks!